regression-test/test_2160.py

```diff
@@ -0,0 +1,32 @@
+#
+# Regression test for continue training on native data
+#
+# https://github.com/RaRe-Technologies/gensim/issues/2160
+#
+import os.path as P
+from gensim.models import FastText
+from gensim.test.utils import common_texts
+
+curr_dir = P.dirname(P.abspath(__file__))
+
+
+def train_gensim():
+    path = P.join(curr_dir, 'toy-data.txt')
```

Comment:
use `datapath` function instead (will work after you place data & FB model in suitable place)